skrub/_table_vectorizer.py

```diff
@@ -567,7 +567,12 @@ def _check_X(self, X):
                     "your data has a single feature or array.reshape(1, -1) "
                     "if it contains a single sample."
                 )
-            feature_names = getattr(self, "feature_names_in_", None)
+            feature_names = getattr(
+                self,
+                "feature_names_in_",
+                # by default, we will use "x0", "x1", ...
+                np.asarray([f"x{i}" for i in range(X_array.shape[1])], dtype=object),
```

Comment:
Because we did not need the feature names then it was maybe fine. But we were having inconsistent names: https://github.com/scikit-learn/scikit-learn/pull/27801

Reply:
Actually, we might have a scikit-learn bug here. I have to check if we should forward the `.set_output` to the underlying `FunctionTransformer` if this is set on the `ColumnTransformer`.

Reply:
This is a bug.